src/lib/mdns/Resolver_ImplMinimalMdns.cpp

```diff
@@ -338,16 +356,20 @@ class MinMdnsResolver : public Resolver, public MdnsPacketDelegate
     }
     static constexpr int kMaxQnameSize = 100;
     char qnameStorage[kMaxQnameSize];
+
+    static Mdns::IPCache<CHIP_CONFIG_IPCACHE_SIZE, CHIP_CONFIG_TTL_MS> sIPCache;
```

Comment:
why static?
`MinMdnsResolver gResolver;` is already global. Let's not have a global inside a global.

Reply:
good point